telemetry/src/main/java/com/newrelic/telemetry/TelemetryClient.java

```diff
@@ -71,26 +71,31 @@ public void sendBatch(SpanBatch batch) {
   private void scheduleBatchSend(
       BatchSender sender,
       TelemetryBatch<? extends Telemetry> batch,
-      int waitTime,
+      long waitTime,
       TimeUnit timeUnit) {
-    executor.schedule(
-        () -> sendWithErrorHandling(sender, batch, waitTime, timeUnit), waitTime, timeUnit);
+    scheduleBatchSend(sender, batch, waitTime, timeUnit, Backoff.defaultBackoff());
```

Comment:
are user-configurable backoff strategies planned for a later PR?

Reply:
there's probably room for interpretation, but I didn't see in the spec that user configured times were required.  In any case, yes, prefer to do that in a future PR.

Reply:
```
For a backoff factor of 1 second, and a backoff max of 16 seconds, the retry delay interval should follow a pattern of [0, 1, 2, 4, 8, 16, 16, ...]. Subsequent retries should wait 16 seconds until the request has been retried the configured max retries number of times.

The total retry duration can be computed from the combination of backoff factor and backoff max. SDKs may provide a function to configure retry behavior by specifying the total retry duration instead of max retries.
```

It certainly mentions "configure" a bunch, but a bit obliquely. We should PR into the specs to make it very clear what should be user-configurable and what the defaults should be.